File: test/Data.Analyzers.Vsix.Test/VisualBasic/MounterNaming.vb
Public Class MounterNaming
    Inherits Model

    Protected Shared ReadOnly _Column1 As Mounter(Of _Int32) = RegisterColumn(Function(x As MounterNaming) x.Column2)
    Protected Shared ReadOnly _Column2 As Mounter(Of _Int32)

    Shared Sub New()
        _Column2 = RegisterColumn(Function(x As MounterNaming) x.Column1)
    End Sub

    Private m_Column1 As _Int32
    Public Property Column1 As _Int32
        Get
            Return m_Column1
        End Get
        Private Set
            m_Column1 = Value
        End Set
    End Property

    Private m_Column2 As _Int32
    Public Property Column2 As _Int32
        Get
            Return m_Column2
        End Get
        Private Set
            m_Column2 = Value
        End Set
    End Property
End Class

File: test/Data.Analyzers.Vsix.Test/CSharp/MounterNaming.cs
namespace DevZest.Data.Analyzers.Vsix.Test.CSharp
{
    public class MounterNaming : Model
    {
        protected static readonly Mounter<_Int32> _Column1 = RegisterColumn((MounterNaming _) => _.Column2);
        protected static readonly Mounter<_Int32> _Column2;

        static MounterNaming()
        {
            _Column2 = RegisterColumn((MounterNaming _) => _.Column1);
        }

        public _Int32 Column1 { get; private set; }

        public _Int32 Column2 { get; private set; }
    }
}
